consult.el

```diff
@@ -1312,5 +1316,33 @@ Prepend PREFIX in front of all items."
 
 (add-hook 'consult-preview-mode-hook #'consult--icomplete-preview-setup)
 
+(defun consult--flymake-candidates ()
+  "Return Flymake errors as alist."
+  (consult--forbid-minibuffer)
+  (let* ((fmt (format "%s" "foo")))
+    (mapcar
+     (lambda (diag)
+       (goto-char (flymake-diagnostic-beg diag))
+       (cons
+        (format "%s %s %s"
```

Comment:
I should propertize this string,.

Reply:
Yes, ideally with the flymake fonts. It would be great if you can align the formatting with the table formatting of the flymake buffer. Please take a look at how the flycheck command does it.

Reply:
Done: flymake fonts are used, and columns are aligned.